### PR TITLE
Fix for #1543 Localization Resources not being loaded for ext Modules

### PR DESCRIPTION
**Pull request by leigh-pointer**
The EnumerateFiles was not using the correct searchPattern, the Wildcard (*) was missing which in turn will not find any local Resources.

### CODE REVIEW

File: Oqtane.Shared/Shared/Constants.cs [hunk 1]
```diff
@@ -72,7 +72,7 @@ public class Constants {
         };
         public static readonly string[] InvalidFileNameEndingChars = { ".", " " };
 
-        public static readonly string SatelliteAssemblyExtension = ".resources.dll";
+        public static readonly string SatelliteAssemblyExtension = "*.resources.dll";
```
Comment (hishamco) [inline Oqtane.Shared/Shared/Constants.cs:75]: I presume this is an extension which is fine, if there's a bug in search we could fix in the place where search occurs

Comment (leigh-pointer) [reply Oqtane.Shared/Shared/Constants.cs:75]: It occurs in `static class OqtaneServiceCollectionExtensions` 
on `static void LoadSatelliteAssemblies(string[] supportedCultures)`

The code in foreach would **never** execute without a wildcard
`foreach (var assemblyFile in assembliesFolder.EnumerateFiles(Constants.SatelliteAssemblyExtension))`

Comment (hishamco) [reply Oqtane.Shared/Shared/Constants.cs:75]: I created #1544 which don't break the satellite extension

**Comment by sbwalker** [comment]
@leigh-pointer I merged #1545 because it preserves the original intent of the SatelliteAssemblyExtension constant - which is to identify the file extension for satellite assemblies. The wildcard makes more sense to be implemented in the method where there the constant is used.

**Pull request closed without merging**